React/Views/RCTWrapperViewController.m

```diff
@@ -93,6 +98,19 @@ - (void)viewWillAppear:(BOOL)animated
   }
 }
 
+- (UIImageView *)findHairlineImageViewUnder:(UIView *)view {
```

Comment:
Formatting is off. Make sure to use 2 space indents and curly braces should be on their own line.